api/python/provisioner/srv/salt/cortx_repos/install.sls

```diff
@@ -17,7 +17,9 @@
 
 # TODO TEST
 
-{% if "RedHat" in grains['os'] %}
+# Note. we use True here since currently we do not consider
+#       any usage of an alternative option
+{% if "RedHat" in grains['os'] or True %}
```

Comment:
I don't understand this. It would always result in `True`. How does that help?

Reply:
That part (RedHat) is used only for local verification cases. The alternative condition just doesn't make sense in our current flow. Generally we should re-write / remove that soon.